.github/workflows/build.yml

```
@@ -633,19 +633,27 @@ jobs:
   flatpak:
     runs-on: ubuntu-latest
     container:
-      image: bilelmoussaoui/flatpak-github-actions:kde-6.7
+      image: ghcr.io/flathub-infra/flatpak-github-actions:kde-6.8
       options: --privileged
     steps:
       - name: Checkout
         uses: actions/checkout@v4
         if: inputs.build_type == 'Debug'
         with:
-          submodules: "true"
+          submodules: true
+
+      - name: Set short version
+        shell: bash
+        run: |
+          git config --global --add safe.directory $GITHUB_WORKSPACE
+          ver_short=`git rev-parse --short HEAD`
+          echo "VERSION=$ver_short" >> $GITHUB_ENV
```

We don't need to run Git for revision information, as we already have it

```suggestion
          echo "VERSION=${GITHUB_SHA::9}" >> $GITHUB_ENV
```

---

Thanks for this. I've used `${GITHUB_SHA::7}` though because all other artifacts also use the first 7 characters of a commit. Hope that's ok...

---

I'm having network issues, apparently. My `git push` is failing atm...

EDIT: Done.